internal/pkg/describe/rd_web_service.go

```diff
@@ -173,3 +225,25 @@ func (w *rdWebSvcDesc) HumanString() string {
 	writer.Flush()
 	return b.String()
 }
+
+func (w *rdWebSvcDesc) hasObservabilityConfiguration() bool {
```

Comment:
can we make this a method of `observabilityPerEnv` instead of `rdWebSvcDesc` so that it can be leveraged by other service descriptions?